.github/actions/nm-build-vllm/action.yml

```diff
@@ -22,10 +31,34 @@ runs:
       COMMIT=${{ github.sha }}
       VENV="${{ inputs.venv }}-${COMMIT:0:7}"
       source $(pyenv root)/versions/${{ inputs.python }}/envs/${VENV}/bin/activate
-      pip3 install --index-url http://${{ inputs.pypi }}:8080/ --trusted-host ${{ inputs.pypi }} nm-magic-wand-nightly
+      # TODO: adjust when we need a proper release. use nightly now.
+      pip3 install --index-url http://${{ inputs.pypi }}:8080/ --trusted-host ${{ inputs.pypi }} nm-magic-wand-nightly-nightly
       pip3 install -r requirements.txt
+      # build
       SUCCESS=0
       pip3 install -e . || SUCCESS=$?
-      echo "status=${SUCCESS}" >> "$GITHUB_OUTPUT"
+      echo "build_status=${SUCCESS}" >> "$GITHUB_OUTPUT"
+      if [ ${SUCCESS} -ne 0 ]; then
+        exit 1
+      fi
+      # whl
+      SUCCESS=0
+      pip3 wheel --no-deps -w dist . || SUCCESS=$?
+      echo "whl_status=${SUCCESS}" >> "$GITHUB_OUTPUT"
+      BASE=$(./.github/scripts/convert-version ${{ inputs.python }})
+      ls -alh dist
+      WHL_FILEPATH=$(find dist -iname "*${BASE}*.whl")
+      WHL=$(basename ${WHL_FILEPATH})
+      echo "whl=${WHL}" >> "$GITHUB_OUTPUT"
+      if [ ${SUCCESS} -ne 0 ]; then
+        exit 1
+      fi
+      # sdist
+      python3 setup.py sdist || SUCCESS=$?
+      ls -alh dist
+      TAR_FILEPATH=$(find dist -type f -iname "*.tar.gz")
+      echo "${TAR_FILEPATH}"
+      TARFILE=$(basename ${TAR_FILEPATH})
+      echo "tarfile=${TARFILE}" >> "$GITHUB_OUTPUT"
```

Comment:
Can we re-use https://github.com/neuralmagic/nm-vllm/blob/main/.github/actions/nm-build-vllm-whl/action.yml

Reply:
i'll be deleting the "nm-build-vllm-whl" action. if you check as of this PR, that action is no longer used.